# HG changeset patch
# User Thomas Klausner <tk@giga.or.at>
# Date 1453987295 -3600
#      Thu Jan 28 14:21:35 2016 +0100
# Node ID 1472ec4a4519a53165a8ce66ffb5d328305509b9
# Parent  25c609cdb41d1963cffff26338ac3b8d47072196
Add 'extern int optopt;' for Linux and perhaps others.

Reported by Remi Collet.

diff -r 25c609cdb41d -r 1472ec4a4519 src/ziptool.c
--- a/src/ziptool.c	Thu Jan 28 09:54:45 2016 +0100
+++ b/src/ziptool.c	Thu Jan 28 14:21:35 2016 +0100
@@ -48,6 +48,7 @@
 #ifndef HAVE_GETOPT
 #include "getopt.h"
 #endif
+extern int optopt;
 
 #include "zip.h"
 #include "compat.h"
